script/Arcueid/Day 5/05_00_ARC05_7_2.txt

```diff
@@ -32,13 +33,13 @@ That was not his abnormality―――
 [sha:91a67e94732760ed733be16c74ad2a52933fbe37]{
 -- Page 514, Offset 9974. Glued. Mods: @x.
 -- 逆なんだ、アルクェイド……！」
- We got it backwards, Arcueid...!"
+ We got it backward, Arcueid...!"
 }
 [sha:dec8b0f01bdd48e2693f8312193b34d74137c388]{
 -- Page 514, Offset 9975.
 -- 地上から、遠く離れた彼女に向けて叫ぶ。
 // Non-literal - 地上から、遠く離れた彼女に向けて叫ぶ。
-From the top of the pit, I call out to her across the expanse.
+From the top of the pit, I call out to her from across the expanse.
```

Comment:
Revert, he's already `from` the edge and his voice has to carry across the expanse